README.md

```diff
@@ -1 +1,35 @@
-# cvs-wms
\ No newline at end of file
+# cvs-wms
+
+Create WMS database locally
+
+## Prerequisites :
+installed Liquibase
+installed MySQL Database Software
+installed MySQL Connector/J (https://dev.mysql.com/doc/connectors/en/connector-j-binary-installation.html)
+
+## Liquibase Steps
+
+### Create configuration file liquibase.properties, provide database name (default: WMS_EDH_STAGING), database username and password and class path to MySQL J/Connector.
+
+```properties
+driver: com.mysql.cj.jdbc.Driver
+url: jdbc:mysql://localhost:3306/CVSBNOP?createDatabaseIfNotExist=true
+username: root
+password: password
+classpath: mysql-connector-java-8.0.23.jar
```

Comment:
I would possibly remove any reference to the Connector/J version, the latest one is .31.

Reply:
I'll make it generic, that its the full path to where ever the jar files lives locally.  I ran it locally and it was happy with the jar file being in my downloads folder - so long as I gave the full path to it